service/worker/scanner/scanner.go

```diff
@@ -117,36 +120,40 @@ func New(
 
 // Start starts the scanner
 func (s *Scanner) Start() error {
-	backgroundActivityContext := context.Background()
+	bac := context.Background()
```

Comment:
Nit: lol, I haven't seen this name as a convention in the code base. I think ctx should be fine, the rest of the code may not care if it's from background or not.